server/permissions/UserRole.ts

```diff
@@ -0,0 +1,14 @@
+export enum UserRole {
+  Admin = "admin",
+  Host = "host",
```

Comment:
these should have the first letter capitalized as they are in the frontend version, i dont think it'll work otherwise

also the name of the enum is `UserRole` here but `UserRoles` on the frontend
should probably rename one or the other to be consistent, i personally prefer `UserRole`

Reply:
Idk how I completely missed the capitalization lol, fixed in 930713372aa37c745478212d76832d6c51374925

Yeah it should probably be `UserRole`, in the frontend I was thinking it more as an array of `UserRole`s, but now that it's an enum, I've renamed it in 82bf7f9222ad9d4bc387487842bf5dff3f646a2a